### PR TITLE
Iteration 1 Whiteboard

### PR DESCRIPTION
**Pull request by ojhstill**
### TP011 - Interactive Whiteboard
**User Stories Complete:** 016
**Authors:** Oliver Still and Cameron Smith

### CODE REVIEW

File: client/src/main/java/application/Launcher.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void start(Stage stage) throws Exception {
          * at the start of the program. For example to skip the login stage and test your own
          * window. */
         ViewFactory viewFactory = new ViewFactory(mainConnection);
-        viewFactory.showLoginWindow();
+        viewFactory.showWhiteboardWindow();
```
Comment (ojhstill) [inline client/src/main/java/application/Launcher.java]: Perhaps change back to `viewFactory.showLoginWindow()`.

**Review by JADGardner — CHANGES_REQUESTED** [review]
Everything looks good to me. Just need to remove the README from the pull request and fix that line in Launcher.java

**Review by dmb537 — CHANGES_REQUESTED** [review]
Looks good yeah - just the previously mentioned change (or rather, not change) to Launcher.java. Don't know how to do that though - might be able to be done during the merging process or something?

**Comment by ojhstill** [comment]
All fixed - Should be good to go now!

**Review by dmb537 — APPROVED** [review]
Yeah looks good now

**Review by dmb537 — APPROVED** [review]

**Review by JADGardner — APPROVED** [review]
All good, approved! 👍

**Pull request merged**